### PR TITLE
Succint Header Change

### PR DESCRIPTION
**Pull request by kroos47**

### CODE REVIEW

**Review by Leouarz — APPROVED** [review]
Seems ok, i will check if it works correctly when other parts are merged

File: packages/apps-config/src/api/spec/dataavail.ts [hunk 1]
```diff
@@ -85,12 +85,12 @@ const definitions: OverrideBundleDefinition = {
...
+          commitment: 'Vec<u8>',
+          dataRoot: 'H256'
         },
         V1HeaderExtension: {
-          commitment: 'KateCommitment',
-          appLookup: 'DataLookup'
+          appLookup: 'DataLookup',
+          commitment: 'KateCommitment'
         },
         VTHeaderExtension: {
```
Comment (Leouarz) [inline packages/apps-config/src/api/spec/dataavail.ts:95]: Should it be changed too ? Just for the sake of completion ?

File: packages/apps-config/src/api/typesBundle.ts [hunk 2]
```diff
@@ -49674,12 +49674,12 @@ export const typesBundle = {
             "KateCommitment": {
               "rows": "Compact<u16>",
               "cols": "Compact<u16>",
+              "commitment": "Vec<u8>",
```
Comment (0xSasaPrsic) [inline packages/apps-config/src/api/typesBundle.ts:49677]: was this missing, I don't see that order was changed?

File: packages/apps-config/src/api/spec/dataavail.ts (hunk 1 above)
Comment (prabal-banerjee) [reply packages/apps-config/src/api/spec/dataavail.ts:95]: Both Kate Commitment and Header Extension needed change? @kroos47

Comment (kroos47) [reply packages/apps-config/src/api/spec/dataavail.ts:95]: Its being done in another place where types are overriden

**Pull request merged**